gradle-hsql-plugin/src/main/groovy/HsqldbStopTask.groovy

```diff
@@ -0,0 +1,32 @@
+package pl.tlempart.gradle.plugins.hsqldb
```

Comment:
Groovy code code should not be there. Plugin is using java code and not groovy code. I accidently pushed the groovy code. It was my mistake. I really apologize for that. I was playing with different  approaches and came to know that java approach works, but I forgot to remove the groovy code.